frontend/src/components/video-chat/video-comment/video-comment.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'common/types/types';
+import { Comment } from 'shared/src/common/types/comment';
+
+import styles from './video-comment.module.scss';
+
+type Props = {
+  comment: Comment;
+};
+
+const VideoComment: FC<Props> = ({ comment }) => {
+  return (
+    <div className={styles['video-comment']}>
+      <div className={styles['main-part-comment']}>
+        <img className={styles['commentators-avatar']} src={comment.avatar} alt={comment.name} />
+        <p className={styles['commentators-name']}>{comment.name}</p>
+        <span className={styles['dispatch-time']}>{comment.dateAdded}</span>
```

Comment:
you would probably have date timestamp from backend. Use dayjs to prepare date formatting https://day.js.org/